src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -20,6 +18,10 @@
     <OutputPath>$(BinDir)IL/</OutputPath>
     <Configurations>Debug;Release;Checked</Configurations>
     <Platforms>x64;x86;arm;arm64</Platforms>
+
+    <ILLinkClearInitLocals>true</ILLinkClearInitLocals>
+    <ILLinkTrimAssembly>true</ILLinkTrimAssembly>
+    <ILLinkTrimXml>unused</ILLinkTrimXml>
```

Comment:
Should this rather point to the auto-generated ILLinkTrimXml under obj directory?

Reply:
You are absolutely right. Pushed a commit.